src/entities/activity/lib/types/conditionalLogic.ts

```diff
@@ -15,10 +15,130 @@ type Condition =
   | EqualCondition
   | NotEqualCondition
   | BetweenCondition
-  | OutsideOfCondition;
+  | OutsideOfCondition
```

Comment:
Nit: Could we keep these and the individual types in both this file & `ActivityItemDto.ts` in the same order? In order to improve readability